From 016fb2e962cea219f5ac6c5dd8312239b3e17d0e Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 18 Sep 2018 11:49:19 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable DEBUG_SET_MODULE_RONX

https://android.googlesource.com/kernel/configs
36b7268642c0 ("add memory-permission related configs")

These configs are already required as part of CTS.

Change-Id: If285686e5a917d68e94303f1485e31ff063ecc82
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 93247f04e20e..8900d82c372e 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -806,6 +806,7 @@ CONFIG_SCHEDSTATS=y
 CONFIG_TIMER_STATS=y
 # CONFIG_DEBUG_PREEMPT is not set
 CONFIG_ENABLE_DEFAULT_TRACERS=y
+CONFIG_DEBUG_SET_MODULE_RONX=y
 CONFIG_KEYS=y
 CONFIG_SECURITY_PERF_EVENTS_RESTRICT=y
 CONFIG_SECURITY=y
-- 
2.35.3

